### ISSUE TITLE
Notes panel should toggle open and closed without losing scroll state

### ISSUE THREAD

**Issue by pablo-mayrgundter** [bldrs-ai/Share#317]
When working through an issue and its comments, I'd like to be able to toggle the notes panel open and closed without resetting scroll state to the top issue.

**Comment by OlegMoshkovich** [comment]
yes good one - keeping track of the scroll position would be cool - seem we would need to keep track of the y coordinate and store it in the state

**Comment by pablo-mayrgundter** [comment]
Maybe prioritize permalink?  I'd start with setting the comment offset in the permalink, and then add fine-grained y-pos to store.

So I guess this is related to #313 and #315.

**Comment by OlegMoshkovich** [comment]
Feature Request.
